starlite/app.py

```diff
@@ -200,6 +200,7 @@ def __init__(
         template_config: Optional["TemplateConfig"] = None,
         type_encoders: Optional["TypeEncodersMap"] = None,
         websocket_class: Optional[Type["WebSocket"]] = None,
+        configuration: Optional[AppConfig] = None,
```

Comment:
```suggestion
        config: Optional[AppConfig] = None,
```

`config` is the canonical way we refer to configuration objects (`AppConfig`, not `AppConfiguration`)